source/interprocedural_analyses/taint/stepLogger.ml

```diff
@@ -0,0 +1,24 @@
+open Core
+
+module StepLogger = struct

```

Comment:
This should not be needed. I think this probably doesn't compile since it doesn't match `stepLogger.mli`
```suggestion
```